odl/test/tomo/geometry/geometry_test.py

```diff
@@ -549,6 +550,48 @@ def test_fanbeam_frommatrix():
                                                    det_rad, sing_mat)
 
 
+def test_fanbeam_flying_focal_spot():
+    """Test the flying focal spot in 2d fan beam geometry."""
+    full_angle = np.pi
+    apart = odl.uniform_partition(0, full_angle, 10)
+    dpart = odl.uniform_partition(0, 1, 10)
```

Comment:
Avoid using the same magic number everywhere in tests (here: 10) since it can hide bugs. It's usually recommended to use different prime numbers.